mu-plugins/global-fonts/index.php

```diff
@@ -69,8 +69,8 @@ function preload_font( $fonts, $subsets ) {
 
 	$preload = $style->extra['preload'] ?? [];
 	$new_preload = [
-		'fonts' => array_unique( $fonts ),
-		'subsets' => array_unique( $subsets ),
+		'fonts' => $fonts,
+		'subsets' => $subsets,
```

Comment:
For the cases:
`global_fonts_preload( 'Inter, Inter', 'Latin, Latin-ext' );`
`global_fonts_preload( 'Inter, EB Garamond', 'Latin, Latin );`

Reply:
Why do you need to specify "Inter" twice, or "latin" twice?

From the current code, I would expect these outcomes:

| Function call | to preload |
|---------------|------------|
| `global_fonts_preload( 'Inter',  'latin' )` | Inter-latin |
| `global_fonts_preload( 'Inter, EB Garamond',  'latin' )` | Inter-latin and EBGaramond-latin |
| `global_fonts_preload( 'Inter',  'latin, cyrillic' )` | Inter-latin and Inter-cyrillic |
| `global_fonts_preload( 'Inter, EB Garamond', 'latin, latin-ext' )` | Inter-latin, Inter-latin-ext, EBGaramond-latin, and EBGaramond-latin-ext |

Maybe requiring just a single font in the first parameter would help, if the aim is to allow for different subsets per font.

Reply:
Because originally the function works like this:
| Function call | to preload |
|---------------|------------|
| `global_fonts_preload( 'Inter',  'latin' )` | Inter-latin |
| `global_fonts_preload( 'Inter, EB Garamond',  'latin' )` | Would fail, because EB Garamond has nothing to pair |
| `global_fonts_preload( 'Inter',  'latin, cyrillic' )` | Would fail, because Cyrillic has nothing to pair |
| `global_fonts_preload( 'Inter, EB Garamond', 'latin, latin-ext' )` | Inter-latin, EBGaramond-latin-ext |

So if we want to preload `Inter latin` and `Inter latin-ext`,  it needs to be `global_fonts_preload( 'Inter, Inter', 'Latin, Latin-ext' );`

I've changed to your idea, it's smarter.
Although there's a case that it needs to call `global_fonts_preload` twice when we want to preload `latin` and `latin-ext` for `Inter` and only `latin` for `EB Garamond`, such a case should never happen as subsets required for a certain page should be the same.